src/repart/repart.c

```diff
@@ -8588,6 +8633,17 @@ static int parse_argv(int argc, char *argv[], X509 **ret_certificate, EVP_PKEY *
                                 return r;
                         break;
 
+                case ARG_APPEND_FSTAB:
+                        if (isempty(optarg)) {
+                                arg_append_fstab = APPEND_AUTO;
+                                break;
+                        }
+
+                        arg_append_fstab = append_mode_from_string(optarg);
+                        if (arg_append_fstab < 0)
+                                return log_error_errno(arg_append_fstab, "Failed to parse --append-fstag parameter: %s", optarg);
```

Comment:
please suffix swicthes that takes arguments with `=` in our log msgs/docs, to indicate they take arguments.

or in other words, write out `--append-fstab=` here

Reply:
done